frontend/public/components/machine-set.tsx

```diff
@@ -268,6 +297,18 @@ export const MachineSetList: React.SFC = (props) => {
         transforms: [sortable],
         props: { className: tableColumnClasses[2] },
       },
+      {
+        title: t('public~Instance type'),
+        props: { className: tableColumnClasses[2] },
```

Comment:
We should create separate `tableColumnClasses` for each row you're adding.

How does this render at different screen widths? Have you tested with the quick start sidebar open?